### ISSUE TITLE
Add PaymentMethodeOrder to stripe checkout?

### ISSUE THREAD

**Issue by bodomalo** [stripe/stripe-php#1625]
### Is your feature request related to a problem? Please describe.

I have limited the payment methodes on strip checkout, my customers decides before(!) on my website what payment methode they wann use (like PAYPAL, BANK TRANSFER, CARD (=stripe), GIROPAY (=stripe), PAYSAFECARD, SOFORT (=stripe)). So I use different providers.

For Stripe checkout I set: 
'payment_method_types' => ['sofort'],
so the customer only has this option.

I see that customers who selects SOFORT and STRUGGLES with the payment, can not SWITCH to CARD then. Usually he is lost and will not pay. (Lost customer)

If I use this-
'payment_method_types' => ['sofort', 'card', 'giropay'],
Then the stripe window will change the order to
[card], [sofort],[giropay] 
Stripe thinks this is ideal.
This is not ideal, because my customers pre-selected SOFORT (on my website)

### Describe the solution you'd like

Solution:
Add the 
PAYMENT_METHODE_ORDER field  ['sofort'], ['card']
And I can pre-select my prefered payment methode, and customer can still switch to CARD if he needs to.

### Describe alternatives you've considered

There is no real alternative... 

### Additional context

The order of payment methodes stripe wants me to use is not perfect when I pre-select a payment methode in my own checkout page

**Comment by remi-stripe** [comment]
@bodomalo Github issues are limited to bugs or feature requests with the stripe-php SDK and this is more an API feature ask better directed at our support team: https://support.stripe.com/contact

We don't have plans to add this to the API for now. Checkout will prioritize the payment methods we think are best adapted for your customer based on where they are paying from. We want to continue improving this over time so that SOFORT is offered before card where it makes the most sense and apply the same to other types! I'll share your example with the product team in charge of evolving Checkout!

**Comment by bodomalo** [comment]
Hello

Since YESTERDAY, I get this error
*The payment method type provided: giropay is invalid. *


I use GIROPAY since 2 years
I did NOT change anything in my configuration.

Stripe support is absolut useless and tells me I should read the manual on
how to integrate API.
Sorry, the message says "GIROPAY is INVALID"...
EPS is valid
CARD is valid
APPLEPAY is valid
all those things work
but only GIROPAY not anylonger? WHY?

Uncaught exception 'Stripe\Error\InvalidRequest' with message 'The payment
method type provided: giropay is invalid. Please ensure the provided type
is activated in your dashboard (
https://dashboard.stripe.com/account/payments/settings) and your account is
enabled for any preview features that you are trying to use.

Am Di., 2. Jan. 2024 um 16:37 Uhr schrieb remi-stripe <
***@***.***>:

**Comment by remi-stripe** [comment]
@bodomalo Please work with our support team for help: https://support.stripe.com/contact

**Comment by bodomalo** [comment]
I am sorry the stripe support is so shitty they do not understand anything.
They do not get what I want, they just send me a link to their help support
pages.

It worked 2 days ago
I did not change anything
Now it says "GIROPAY" is not activated.

And they send me stupid answers about how I set up giropay. Me nothing
doing wrong.
Why is the freakign support on stripe so stupid bad?



Am Mo., 24. Juni 2024 um 20:26 Uhr schrieb remi-stripe <
***@***.***>:
